crates/motionfile/src/motion_interpolator.rs

```diff
@@ -142,7 +156,7 @@ impl<T: Debug + Interpolate<f32>> MotionInterpolator<T> {
                 if time_since_start >= current_frame.spline.total_duration() {
                     State::CheckExit {
                         current_frame_index,
-                        time_since_start: Duration::ZERO,
+                        time_since_start: current_frame.spline.total_duration(),
```

Comment:
you cannot do that. This will break the timeout functionality of conditions.

Reply:
Has been undone